app/views/editor/posts/_form.html.haml

```diff
@@ -23,7 +23,13 @@
       .links
         = link_to_add_association 'Add credit', f, :credits, class: 'btn btn-default pull-right'
 
-  = f.collection_select :category_id, current_site.categories.leaves, :id, :full_name, {}, {class: "select2", "data-select2-allow-clear": false}
+  = f.form_group label: {text: :categorizations}, help: "カテゴリを追加します" do
```

Comment:
* There are no need to use Symbol for text.
* This text should be a "Categories", not a "Categorizations".

``` diff
- {text: :categorizations}
+ {text: 'Categories'}
```